clash/bloody_grail.tex

```diff
@@ -29,25 +29,24 @@ \subsection*{\MakeUppercase{Player Setup}}
 
 \textbf{Map Tile Pool:} Each player takes 2 random Far (II--III) Map Tiles, one of which must contain a Settlement
 
-\textbf{Additional Bonus:} None
-
 \subsection*{\MakeUppercase{Map Setup}}
 Take the following Map Tiles and arrange them as shown in the Scenario map layout:
+\vspace*{\fill}\columnbreak
```

Comment:
I'm not sure if that's the best place for the split. You have a colon, and then a big blank space. A line ending with a colon implies looking below it. Maybe you could put the columnbreak below the first bullet point which fits entirely in the left column?